### ISSUE TITLE
LEARNING RATE CHANGES FROM 2e to 9e at HALF training - IS THIS NORMAL?

### ISSUE THREAD

**Issue by LIQUIDMIND111** [TheLastBen/fast-stable-diffusion#471]
hello BEN, any ideas and are you aware that learning rate CHANGES drastically during training? 
it goes from **2e** up to **9e** at half training.... look in BOLD

_"""Training the unet...
Progress:|██████████████ | 54% 1608/3000 [25:16<21:40, 1.07it/s, loss=0.0192, **lr=9.82e-7**] uvtbehrtangxie
"""_

is this normal??? 

seems an error that is making results bad....?? can you confirm? 

It comes back down at 90% of training completion. you can see also my instance name is NORMAL as requested... (uvtbehrtangxie) so if I get BAD outputs.. Could it be this learning rate discrepancy??? 

I dont remember SEEN learning rates go SO HIGH or even change during past trainings.... this is new to my eyes.....

**Comment by LIQUIDMIND111** [comment]
now at 78% training is going back down....

Training the unet...
'########:'########:::::'###::::'####:'##::: ##:'####:'##::: ##::'######:::
... ##..:: ##.... ##:::'## ##:::. ##:: ###:: ##:. ##:: ###:: ##:'##... ##::
::: ##:::: ##:::: ##::'##:. ##::: ##:: ####: ##:: ##:: ####: ##: ##:::..:::
::: ##:::: ########::'##:::. ##:: ##:: ## ## ##:: ##:: ## ## ##: ##::'####:
::: ##:::: ##.. ##::: #########:: ##:: ##. ####:: ##:: ##. ####: ##::: ##::
::: ##:::: ##::. ##:: ##.... ##:: ##:: ##:. ###:: ##:: ##:. ###: ##::: ##::
::: ##:::: ##:::. ##: ##:::: ##:'####: ##::. ##:'####: ##::. ##:. ######:::
:::..:::::..:::::..::..:::::..::....::..::::..::....::..::::..:::......::::

Progress:|████████████████████     | 79% 2360/3000 [37:00<09:56,  1.07it/s, loss=0.00239, lr=5.06e-7] bvuwyfjennxiffer   uvtbehrtangxie

**Comment by TheLastBen** [comment]
This is how the polynomial scheduler works, it avoids overfitting and is compatible with euler a sampler

**Comment by LIQUIDMIND111** [comment]
> This is how the polynomial scheduler works, it avoids overfitting and is compatible with euler a sampler

ok

**Comment by LIQUIDMIND111** [comment]
> bvuwyfjennxiffer uvtbehrtangxie

BRO!!! The results using text encoder at 100, are amazing!!! 

If i use only ONE instance like "a photo bvuwyfjennxiffer in the park" i will get a picture exactly as she is, and if i use the instance a photo of uvtbehrtangxie in the couch", i will get exactly as she is.

**BUT NOW I HAVE AN ISSUE.** IF I USE BOTH, as i have done before with PERFECT RESULTS, like **"a photo of uvtbehrtangxie, and bvuwyfjennxiffer in the park**" - instead of having 2 people with its own faces, i will have 2 girls with a BLEND of each of their faces... this has not happened before on the OLD model when i prompt 2 instances at once..... 

i did this before and got great results.... what do you think is happening now??? i can send screenshots of results if you want...... please let me know whats happening,,,,,
